accio-main/src/main/java/io/accio/main/connector/bigquery/BigQueryMetadata.java

```diff
@@ -155,13 +153,7 @@ public List<String> listFunctionNames(String schemaName)
         if (routines == null) {
             throw new AccioException(NOT_FOUND, format("Dataset %s doesn't contain any routines.", dataset.get().getDatasetId()));
         }
-        return Streams.stream(routines).map(routine -> routine.getRoutineId().getRoutine()).map(routine -> {
```

Comment:
Change to return the real function name in the data source. It may impact the PG required function checking.
https://github.com/Canner/accio/blob/a8bf0477e7406e24f3fdbefb6b612039e08c528e/accio-main/src/main/java/io/accio/main/pgcatalog/PgCatalogManager.java#L168-L170